app/models/vendor.rb

```diff
@@ -0,0 +1,17 @@
+class Vendor < ActiveRecord::Base
+  belongs_to :market
+  has_many :products
+  has_many :sales
+
+  def self.current_month
+    Time.now.month
+  end
```

Comment:
So would the sales_helper.rb file be a better place for this because sales are the objects with the dates? Or a custom helper file like the datetime_helper.rb we made for formatting dates in the views? I'm definitely still fuzzy on the place of helper files.